Quad4.cs

```diff
@@ -0,0 +1,619 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ISAAR.MSolve.PreProcessor.Embedding;
+using ISAAR.MSolve.PreProcessor.Interfaces;
+using ISAAR.MSolve.Matrices.Interfaces;
+using System.Runtime.InteropServices;
+using ISAAR.MSolve.Matrices;
+using ISAAR.MSolve.PreProcessor.Elements.SupportiveClasses;
+
+namespace ISAAR.MSolve.PreProcessor.Elements
+{
+	public class Quad4 : IStructuralFiniteElement, IEmbeddedHostElement
+	{
+		protected static double determinantTolerance = 0.00000001;
+
+		protected static int iInt = 2;
```

Comment:
This should be changed and input to the constructor (maybe have multiple constructors with one of them having the default gauss integration order)